src/software/firmware/srcs/pressure_controller.cpp

```diff
@@ -346,11 +351,16 @@ void PressureController::safeguardPressionCrete(uint16_t p_centiSec) {
 
 void PressureController::safeguardPressionPlateau(uint16_t p_centiSec) {
     if (m_subPhase == CycleSubPhases::HOLD_INSPIRATION) {
-        if (m_pressure >= m_maxPlateauPressureCommand) {
-            // TODO vérifier avec médical si on doit délester la pression
-            // TODO alarme franchissement plateau haut
+        if (m_pressure < ALARM_THRESHOLD_PLATEAU_UNDER_2_CMH2O) {
+            m_alarmController.detectedAlarm(24u, m_cycleNb);
         } else {
-            plateau();
+            m_alarmController.notDetectedAlarm(24u);
+        }
+
+        if (m_pressure > ALARM_THRESHOLD_PLATEAU_ABOVE_80_CMH2O) {
```

Comment:
Is the requirement strict to justify that the alarm is triggered if the pressure is high enough for one iteration during the same cycle. Wouldn't it be better proceeding a mean over a cycle?

Reply:
As I understand, 80cmH2O is a really high pressure for lungs. Especially during plateau subphase. 
I ask the PO and we can't wait for this high pressure.